pyvista/core/filters.py

```diff
@@ -1189,8 +1190,8 @@ def glyph(dataset, orient=True, scale=True, factor=1.0, geom=None,
         absolute : bool, optional
             Control if ``tolerance`` is an absolute distance or a fraction.
 
-        clamping: bool
-            Turn on/off clamping of "scalar" values to range.
+        clamping: bool, optional
+            Turn on/off clamping of "scalar" values to range. Default ``False``.
```

Comment:
The default is in the function signature, although I think it is fine to include here too in general.  But there is no consistency in the documentation on this.  It isn't included in `absolute` parameter either.

Reply:
We've been adding these back in in a lot of places in recent PRs, so I think there's an implicit assumption that defaults should be specified. @akaszynski is my impression correct?

Reply:
When in doubt, I look to see what the big three are doing (`numpy`, `scipy`, and `pandas`).  

Seems like they specify the default in the docstring, *and* and sometimes in the parameter signature.  See:
https://numpy.org/doc/stable/reference/generated/numpy.zeros.html

Given that it's not consistent, I think including it either in the parameter description or signature works for me.  For example, you can look at the pandas docs and it's not specified there consistently either:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rolling.html

I will say that numpydoc does provide guidance here:
https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Where basically all three are fine.  So, it's up to you.  If we do provide a default, just make it literal if it's a keyword so the formatting looks nice.